ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/InMemoryLedgerFactory.scala

```diff
@@ -50,8 +52,8 @@ private[memory] class InMemoryLedgerFactory(dispatcher: Dispatcher[Index], state
     )
   }
 
-  override def ledgerConfig(config: Config[Unit]): LedgerConfiguration =
-    LedgerConfiguration.defaultLocalLedger
+  override def initialLedgerConfig(config: Config[Unit]): InitialLedgerConfiguration =
+    super.initialLedgerConfig(config).copy(delayBeforeSubmitting = Duration.ofMillis(500))
```

Comment:
can we extract the duration into a variable and add a comment as to why we've chosen this duration

Reply:
Yeah, let's do it. I'd like to keep it KV-specific but I guess that works.

Reply:
Done, but I kept it in the KV `Config` object, so other ledgers will have to make their own decisions. I think this is safer than just having one default for all ledgers.